src/scenes/home/informationForm/informationForm.js

```diff
@@ -16,6 +16,7 @@ class SignupInformation extends Component {
     super(props);
     this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
     this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.previousPage = this.previousPage.bind(this);
```

Comment:
` if (this.state.step < 1) {
      return;
    }`

in the previous step method of info form